src/api/dropout.jl

```diff
@@ -130,6 +130,7 @@ end
 @inline _dropout_fptype(x) = float(real(eltype(x)))
 
 CRC.@non_differentiable _dropout_fptype(::Any...)
+EnzymeRules.inactive(::typeof(_dropout_fptype), ::Any...) = nothing
 
```

Comment:
can you see if it still works if you change to `inactive_noinl` for all of these?

It will be performant if so

Reply:
I think some of these are not needed at all. Is there a cost of defining these functions?

Reply:
inactive_noinl is strictly beneficial. It adds inactive metadata to the call/body which can improve performance or do nothing in the worst case if Enzyme can already prove it inactive on own.

inactive itself will stop inlining so there's a potential performance penalty. It will, however, guarantee that the activity information is propagated (whereas the top is presently a best effort procedure).